model/src/order.rs

```diff
@@ -19,6 +19,16 @@ use web3::{
     signing::{self, Key, SecretKeyRef},
     types::Recovery,
 };
+
+/// An order returned by the `solvable_orders` route.
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SolverOrder {
+    #[serde(flatten)]
+    pub order_creation: OrderCreation,
+    pub executable_amount: Option<U256>,
```

Comment:
I assume for partial orders we need to differentiate how much fee has already been paid (how much of sellAmount - executable amount is due to prior fills and how much is due to missing balance) in order to compute the correct fees/discounts.

Reply:
Whether the executable amount is limited due to balance or previous executoins doesn't matter to the solver right? That was my thinking for having only one field. For every execution the fee the contract can take is `order.feeAmount * executedSellAmount / order.sellAmount` so it doesn't depend on previous executon fees.

Reply:
Yes you are right, I thought the feeDiscount depended on previously charged fees, but it only depends on the executed amount of the current trade.